internal/runbits/org/org.go

```diff
@@ -0,0 +1,63 @@
+package org
+
+import (
+	"strings"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+var ErrNoOwner = errs.New("Could not find organization")
+
+type configurer interface {
+	GetString(string) string
+}
+
+// Get returns the name of an organization/owner in order of precedence:
+// - Returns the normalized form of the given org name.
+// - Returns the name of the most recently used org.
+// - Returns the name of the first org you are a member of.
+func Get(desiredOrg string, auth *authentication.Auth, cfg configurer) (string, error) {
+	orgs, err := model.FetchOrganizations(auth)
+	if err != nil {
+		return "", errs.Wrap(err, "Unable to get the user's writable orgs")
+	}
+
+	// Prefer the desired org if it's valid
+	if desiredOrg != "" {
+		// Match the case of the organization.
+		// Otherwise the incorrect case will be written to the project file.
+		for _, org := range orgs {
+			if strings.EqualFold(org.URLname, desiredOrg) {
+				return org.URLname, nil
+			}
+		}
+		// Return desiredOrg for error reporting
+		return desiredOrg, ErrNoOwner
```

Comment:
I believe that in go if an error is returned the other return value is not to be trusted. I think a better approach here may be to use an error type and have the desired org be a member of it.

Reply:
Technically, fixing this is out of scope for the ticket (this is not new code), but I've gone ahead and done so.

Reply:
Thank you! 🙏